datadog_lambda/tracing.py

```diff
@@ -402,6 +546,7 @@ def create_function_execution_span(
             else None,
             "datadog_lambda": datadog_lambda_version,
             "dd_trace": ddtrace_version,
+            "span.name": "aws.lambda",
```

Comment:
Why do you need this? Doesn't `span = tracer.trace("aws.lambda", **args)` (later in this function) set the span name to `aws.lambda`?